### PR TITLE
[pt] Added words to added.txt and spelling.txt

### PR DESCRIPTION
**Pull request by marcoagpinto**
Heya,

Just some missing words.

### CODE REVIEW

File: languagetool-language-modules/pt/src/main/resources/org/languagetool/resource/pt/added.txt [hunk 1]
```diff
@@ -9160,3 +9160,13 @@ ruminais	ruminal	AQ0CP0
 sentenciais	sentencial	AQ0CP0
 grupais	grupal	AQ0CP0
 durais	dural	AQ0CP0
+abébia	abébia	NCFS000
+abébias	abébia	NCFP000
+alónimo	alónimo	AQ0MS0
+alónimos	alónimo	AQ0MP0
```
Comment (p-goulart) [inline languagetool-language-modules/pt/src/main/resources/org/languagetool/resource/pt/added.txt:9166]: Also needs 'alônimo'

File: languagetool-language-modules/pt/src/main/resources/org/languagetool/resource/pt/hunspell/spelling.txt [hunk 2]
```diff
@@ -36,6 +36,20 @@
 #  - ROMAN NUMERALS
 #
 #
+abébia
+abébias
+alónimo
+alónimos
+epitetado
+epitetados
```
Comment (p-goulart) [inline languagetool-language-modules/pt/src/main/resources/org/languagetool/resource/pt/hunspell/spelling.txt:44]: Also needs the feminine forms

File: languagetool-language-modules/pt/src/main/resources/org/languagetool/resource/pt/hunspell/spelling.txt [hunk 3]
```diff
@@ -36,6 +36,20 @@
...
+calçadão
+idadista
+idadistas
+modernaço
+modernaços
```
Comment (p-goulart) [inline languagetool-language-modules/pt/src/main/resources/org/languagetool/resource/pt/hunspell/spelling.txt:49]: Also needs feminine forms

**Comment by marcoagpinto** [comment]
@p-goulart 

Here it is!

**Review by p-goulart — APPROVED** [review]

**Pull request merged**